src/pages/EditMonitor.vue

```diff
@@ -1066,6 +1177,10 @@ message HealthCheckResponse {
 
         },
 
+        addKafkaProducerBroker(newBroker) {
+            this.monitor.kafkaProducerBrokers.push(newBroker);
+        },
+
```

Comment:
I think inlinig this would be benefitial

Reply:
I have tried that using `monitor.kafkaProducerBrokers.push` as `@tag` prop, but I got some weird errors